event_model/__init__.py

```diff
@@ -22,14 +22,21 @@
 from packaging import version
 from pkg_resources import resource_filename as rs_fn
 
-from ._version import get_versions
-
 if sys.version_info < (3, 8):
-    from importlib_metadata import metadata
+    from importlib_metadata import metadata, version  # noqa
 else:
-    from importlib.metadata import metadata
+    from importlib.metadata import metadata, version  # noqa
+
+__version__ = version("event-model")
```

Comment:
I would put the del here too:

```suggestion
__version__ = importlib_version("event-model")
del importlib_version
```